internal/poller/gotimestatus/gotimestatus_test.go

```diff
@@ -0,0 +1,131 @@
+package gotimestatus
+
+import (
+	"bytes"
+	"context"
+	"github.com/rs/zerolog"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+func TestGoTimeStatus_Poll(t *testing.T) {
+	t.Cleanup(func() {
+		timeNow = time.Now
+	})
+	timeNow = func() time.Time {
+		now, _ := time.Parse(time.RFC3339, "2022-11-24T15:20:00Z")
+		return now
+	}
```

Comment:
```go
	now, err := time.Parse(time.RFC3339, "2022-11-24T15:20:00Z")
	if err != nil {
		t.Fatalf("%s", err)
	}

	timeNow = func() time.Time {
		return now
	}

Reply:
I don't see how this could ever fail though; seems unnecessary?